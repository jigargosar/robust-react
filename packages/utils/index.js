export * from './src/hyper-script-utils'
