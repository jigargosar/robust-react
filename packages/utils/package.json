{
  "name": "@jigargosar/utils",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "hyperscript-helpers": "3.0.3",
    "react-hyperscript": "3.2.0"
  }
}
