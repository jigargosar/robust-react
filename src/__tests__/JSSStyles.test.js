// import {SheetsRegistry} from 'jss'
import {withStyles} from 'material-ui'
import {compose} from 'ramda'
import {render} from 'react-testing-library'
import {div, h} from '@jigargosar/utils'

// const sheets = new SheetsRegistry()

const Button = compose(
  withStyles({
    root: {
      color: 'green',
    },
  }),
)(({classes}) => div({className: classes.root}, 'OK'))

it('should render with styles', () => {
  const {container} = render(h(Button))
  // console.log(container)
  expect(container.firstChild).toMatchSnapshot('foo')
})
