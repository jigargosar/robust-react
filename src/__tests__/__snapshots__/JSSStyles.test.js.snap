// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render with styles: foo 1`] = `
<div
  class="Component-root-1"
>
  OK
</div>
`;
